frontend/src/components/table/schedule/sections/base-shifts-section/base-shifts-section.component.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { DataRow } from "../../../../../logic/real-schedule-logic/data-row";
+import { BaseCellComponent } from "../../schedule-parts/base-cell.component";
+import { BaseSectionOptions } from "../base-section/base-section.options";
+import {ShiftRowComponent} from "../../schedule-parts/shift-row.component";
+
+export function BaseShiftsSectionComponent({
+  data = [],
+  onSectionUpdated,
+  logic,
+  metaDataLogic,
+  cellComponent = BaseCellComponent,
+}: BaseSectionOptions) {
+  function onRowUpdated(row: DataRow) {
+    const isSuccess = logic?.tryUpdate(row);
+    if (isSuccess && logic) {
+      onSectionUpdated(logic.sectionData);
+    }
+  }
+
+  return (
+    <React.Fragment>
+      {data.map((dataRow, index) => (
+        <ShiftRowComponent
```

Comment:
Czy my nie możemy przekazywać `ShiftRowComponent` podobnie jak my to robimy z `ShiftCellComponent` ?